repose-aggregator/functional-tests/spock-functional-test/src/test/groovy/framework/mocks/MockValkyrie.groovy

```diff
@@ -101,11 +103,13 @@ class MockValkyrie {
 
         if (method == "GET") {
             if (!missingRequestHeaders) {
-
                 def match = (requestPath =~ permissionsRegex)
                 def tenant = match[0][1]
-                def contact = match[0][2]
+                def call = match[0][2]
+                def contact = match[0][3]
                 contact_id = contact
+                tenant_id = tenant
+                calltype = call
                 _authorizeCount.incrementAndGet()
                 return authorizeHandler(tenant, contact, request)
```

Comment:
Why are we defining these to turn around and only use them to set instance variables?
Isn't this more correct:
```
def match = (requestPath =~ permissionsRegex)
tenant_id = match[0][1]
calltype = match[0][2]
contact_id = match[0][3]
_authorizeCount.incrementAndGet()
return authorizeHandler(tenant_id, contact_id, request)
```